packages/react-select/src/animated/transitions.tsx

```diff
@@ -72,93 +66,62 @@ interface CollapseProps {
   in?: boolean;
   onExited?: ExitHandler<undefined | HTMLElement>;
 }
-interface CollapseState {
-  width: Width;
-}
 
 // wrap each MultiValue with a collapse transition; decreases width until
 // finally removing from DOM
-export class Collapse extends Component<CollapseProps, CollapseState> {
-  duration = collapseDuration;
-  rafID?: number | null;
-  state: CollapseState = { width: 'auto' };
-  transition: { [K in TransitionStatus]?: CSSProperties } = {
-    exiting: { width: 0, transition: `width ${this.duration}ms ease-out` },
-    exited: { width: 0 },
-  };
-  nodeRef = createRef<HTMLDivElement>();
+export const Collapse = ({ children, in: _in, onExited }: CollapseProps) => {
```

Comment:
I assume that `in` is from "react-transition-group"—reserved word for a prop was such a bad idea...

Reply:
agreed 🥲